etc/system-contracts/package.json

```diff
@@ -13,7 +13,7 @@
     "zksync-web3": "^0.13.0"
   },
   "devDependencies": {
-    "@matterlabs/hardhat-zksync-solc": "^0.3.15",
+    "@matterlabs/hardhat-zksync-solc": "^0.4.2",
```

Comment:
I believe you'd also need to update `hardhat.config.ts` due to the following error: `Deprecated zksolc compiler version in hardhat config file (1.3.7). Please use versions 1.3.13 to 1.3.14.`

One thing to note, this directory is coming from https://github.com/matter-labs/era-system-contracts . So we can update for ourselves temporarily, but we should also consider fixing upstream at https://github.com/matter-labs/era-system-contracts/blob/main/package.json#L16